app/src/main/res/values-fr/strings.xml

```diff
@@ -79,16 +79,16 @@
     <string name="invalid_address">Adresse invalide</string>
     <string name="no_file_name">Pas de nom de fichier</string>
     <string name="contact_already_exists">Le contact existe déjà</string>
-    <string name="enter_contact_name">Enter contact name.</string>
-    <string name="wrong_password">Mauvais mot de passe</string>
-    <string name="contact_name_conflict">A contact with the same name exists.</string>
-    <string name="contact_key_conflict">A contact with the same key exists.</string>
-    <string name="contact_name_empty">Contact name is empty.</string>
-    <string name="contact_name_exists">A contact with that name exists already.</string>
-    <string name="contact_remove">Remove contact?</string>
+    <string name="enter_contact_name">Entrez le nom du contact.</string>
+    <string name="wrong_password">Mot de passe non valide</string>
+    <string name="contact_name_conflict">Un contact ayant le même nom existe déjà.</string>
+    <string name="contact_key_conflict">Un contact ayant la même clé existe déjà.</string>
+    <string name="contact_name_empty">Le nom du contact est vide.</string>
+    <string name="contact_name_exists">Un contact ayant ce nom existe déjà.</string>
+    <string name="contact_remove">Supprimer le contact \?</string>
```

Comment:
Is it normal in french to put a slash in front of the question mark?

Reply:
@mwarning I'm using my own Weblate as a translation platform. If it is added, it mean that there's a reason somewhere related to the xml syntax. In my original strings, I have no slash :)